Automattic/wp-calypso
# A4A: Intercept Activity Log links and redirect it to the Preview Pane

Resolve: https://github.com/Automattic/automattic-for-agencies-dev/issues/226

## Proposed Changes

This PR intercepts this route `/activity-log/:site`, and it'll redirect it to `/sites/overview/:site/jetpack_activity`

![image](https://github.com/Automattic/wp-calypso/assets/9832440/09719fb8-dc22-4c11-8109-9c1cdba69283)

**Important Note:** It's working as expected, it intercepts the link and redirect to the correct route, but because of this known issue [The Scan route to /jetpack_scan/history doesn't work on initial state](https://github.com/Automattic/automattic-for-agencies-dev/issues/227) with the initial state, our code is removing the `jetpack_activity` part.

Once it's resolved, it will work well. 

## Testing Instructions

- Check the code
- Localize a site that has the submenu with the View Activity menu. In trunk, you will get a not found page, in this branch, you will redirected to the route `/sites/overview/:site/jetpack_activity` (you will see it for a second), and then it won't open the correct tab (the known issue that will be addressed in another PR)

## Pre-merge Checklist
